db/model/bot.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/bot.js
```

Comment:
Ideally you'd have a fairly verbose description in this comment block about what this is (for all files you're adding)